client/stylesheets/_toCat.scss

```diff
@@ -3770,8 +3775,8 @@ On click the images will be displayed at normal size to complete the effect
 }
 
 .profileUserLeft {
-    margin-left: 9%;
-    width: 35%;
+    margin-left: 4%;
+    width: 40%;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered margin-left, text-align, vertical-align, width